### PR TITLE
Configure Easy Auth for APIM and State ETL Functions

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Adding an easy auth between the states apim endpoints and the states etl function

## Why?

NAC-1137

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [x] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: iac/configure-easy-auth.bash [hunk 1]
```diff
@@ -339,9 +339,15 @@ main () {
     "$STATES_API_APP_ROLE" \
     "$query_tool_identity"
 
-  echo "Configure Easy Auth for all States: ETLStatus and QueryApp"
+  echo "Configure Easy Auth for all States: ETLStatus and QueryApp & StatesApi and States ETL"
```
Comment (bgfoshay) [inline iac/configure-easy-auth.bash:342]: Question: Is "StatesApi" right? I believe this block is configuring EasyAuth between Dashboard and State ETL Functions & configuring EasyAuth between APIM and State ETL Functions

Comment (ryangeorgeharvey) [reply iac/configure-easy-auth.bash:342]: Similar comment to Bill. The comment is also confusing with multiple 'and' and '&' usage.

File: iac/configure-easy-auth.bash [hunk 2]
```diff
@@ -339,9 +339,15 @@ main () {
...
   for etl_app_name in "${etl_function_names[@]}"
     do
+      etl_app_name="${etl_app_name/$'\r'/}"
```
Comment (ryangeorgeharvey) [inline iac/configure-easy-auth.bash:345]: Nitpick (non-blocking): Line 345 is a duplicate of Line 351. Only needed once for the same loop.

**Review by ryangeorgeharvey — APPROVED** [review]
LGTM

**Pull request merged**